tailcfg/tailcfg.go

```diff
@@ -1725,6 +1725,7 @@ const (
 	CapabilitySSHRuleIn          = "https://tailscale.com/cap/ssh-rule-in"           // some SSH rule reach this node
 	CapabilityDataPlaneAuditLogs = "https://tailscale.com/cap/data-plane-audit-logs" // feature enabled
 	CapabilityDebug              = "https://tailscale.com/cap/debug"                 // exposes debug endpoints over the PeerAPI
+	CapabilitySSHAggregator      = "https://tailscale.com/cap/ssh-aggregator"        // enable SSH session aggregation
```

Comment:
I think this should be `ssh-aggregator` based on https://tailscale.slack.com/archives/CQQ1AUCCE/p1674669132275479?thread_ts=1674662145.743249&cid=CQQ1AUCCE

cc @bradfitz

Reply:
I'll also rename it to `NodeAttrSSHAggregator` since we've started prefixing them with `NodeAttr` instead of `Capability`.